<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>291</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Final Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="Slider_FP">
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinFP">
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>500</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Overall Grade: </string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>321</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <spacer name="verticalSpacer_10">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Midterm</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QSpinBox" name="spinFinal">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QSlider" name="Slider_Final">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSlider" name="Slider_Mid">
       <property name="maximum">
        <number>37</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="spinMid">
       <property name="maximum">
        <number>37</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIC 10C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PIC 10B</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="grade">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>500</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>251</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Course:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>81</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>HW 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Slider_HW1">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinHW1">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>HW 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Slider_HW2">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinHW2">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>HW 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Slider_HW3">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinHW3">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>HW 4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Slider_HW4">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinHW4">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>HW 5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Slider_HW5">
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinHW5">
         <property name="maximum">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Course Grade</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>501</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Scheme  1: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_11">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Scheme 2: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_12">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Slider_HW1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinHW1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHW1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_HW1</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_HW2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinHW2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHW2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_HW2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_HW3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinHW3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHW3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_HW3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_FP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinFP</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>423</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinFP</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_FP</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_Final</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinFinal</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinFinal</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_Final</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinMid</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_Mid</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_Mid</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinMid</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>406</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_HW4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinHW4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>304</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHW4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_HW4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_HW5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinHW5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinHW5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Slider_HW5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
